lib/membrane_ffmpeg_swscale/scaler/scaler.ex

```diff
@@ -59,7 +59,7 @@ defmodule Membrane.FFmpeg.SWScale.Scaler do
 
   @impl true
   def handle_process(:input, _buffer, _context, %{native_state: nil} = _state) do
-    raise(RuntimeError, "uninitialized state: Scaler did not receive stream_format")
+    raise "uninitialized state: Scaler did not receive stream format"
   end
```

Comment:
You haven't resolved https://github.com/membraneframework/membrane_ffmpeg_swscale_plugin/pull/17#discussion_r1039841910